### ISSUE TITLE
Config preset (advanced_flybywire_config_v21.xml) is overwritten with default file.

### ISSUE THREAD

**Issue by Triscus** [AlexanderDzhoganov/ksp-advanced-flybywire#34]
The preset file is overwritten with the default file when:

- Starting a new game (even with the same save) and opening the AFBW-config panel
- Reverting a flight and open the AFBW- config panel after then.

No issues when using the config panel in the same 'session'.

**Comment by errolkokbas** [comment]
I can contribute that I did check the config panel each time I tried reloading the game in order to check and see if my presets were saved....

I will try again with this mod and report back after reloading my game and not opening the config panel. 

Either way though, not a workable solution, as this means it becomes worse then trying to make small adjustments in the stock system, instead of having to exit out to home screen, each small change requires complete re-mapping!

**Comment by ddaf** [comment]
That's a very critical bug. Currently I'm unable to reproduce this on my machine. So if you can give any other info about which devices you are using with AFBW and if you see anything suspicious in the log (alt+f2), it would help us a lot to fix this.

**Comment by errolkokbas** [comment]
I'm using a dualshock 4 controller with the DS4Windows x-input wrapper. I'm not flying any missions tonight, but will report back when I have a chance to repro this again and check my log.

**Comment by Triscus** [comment]
I'm using a Saitek X55 Rhino Stick and Throttle. (2 different devices) on Windows 8.1 64bit. They are connected over an USB 3.0 Hub with external power. 

Tonight I wanted to test a bit, and I reinstalled KSP without any other addons. Everything was fine, all devices detected (2 x Saitek and 1x vJoy device) and I switched a couple of times back and forth from flight scene to VAB to Main Menu etc. and the configuration wasn't overwritten. 

But after I restarted KSP, the devices are no longer detected, the config file still exists though. I guess this is the already known issue with Windows 8.1. 

Beside that, it is possible that other Addons are interfering? 

Here are the mode that were installed, while the config file was overwritten.

@errolkokbas: Do you have simliar mods installed?

ALCOR
Kerbal Alarm Clock
Kerbal Engineer
Kethane
Module Manager
Planet Shine
RasterProp Monitor
SCNSat
Telemachus
Texture Replacer
VVRPM

**Comment by ddaf** [comment]
Yeah, sadly our input library is not working very well with Windows 8 at the moment. We'll hopefully find a permanent fix for that in due time. I'll look into if some of those other mods might be conflicting with AFBW.

**Comment by Triscus** [comment]
No, KSP 0.90 32bit

**Comment by DeamonSix** [comment]
Same problem here. Works fine for the current flight, then overwrites with a default after reverting/ending the flight. (Actually I can only confirm that the config is gone when a new flight is started - not sure when it was overwritten.)

This doesn't always happen right away - I can occasionally start and end a few flights. When it does happen though, the game must be restarted. (I tried copying a saved config but it was quickly overwritten)

I tried setting the config to read-only but that just caused a bizarre crash at the start of a flight.

I am also experiencing a bug where my X-55 is not being detected.

XBox360 wireless controller
Win 8.1 Pro x64
KSP 0.90 x86
AFBW 1.4.6 x86 (via ckan)

https://www.dropbox.com/s/xia2x8q7yst1zu9/advanced_flybywire_config_v21.xml?dl=0
https://www.dropbox.com/s/2t6xraiel8tcv9w/output_log.txt?dl=0

![ksp_gamedata_20150201](https://cloud.githubusercontent.com/assets/10803443/5995426/b9918a8e-aa60-11e4-9c7a-f15bb6be1c2d.png)
